src/settings.ini

```diff
@@ -2,8 +2,8 @@
 # Layout: neo, adnw, adnwzjf, bone, koy, kou, vou, qwertz
 layout=neo
 
-# use quote/� as right level 3 modifier
-# �-Taste als rechten Ebene3-Modifier verwenden
+# use quote/ä as right level 3 modifier
```

Comment:
What have you changed here? I don't see any difference. Is it the file encoding?

Reply:
da war glaub echt was anders am file encoding. So wies jetzt ist, stimmt es glaub

Reply:
it showed `�` for all the umlaute for me, so I just added the Umlaute again by keyboard (and something changed, I guess :shrug:)